app/models/vim_performance_daily.rb

```diff
@@ -25,7 +25,7 @@ def self.excluded_cols_for_expressions
   def self.find(cnt, *args)
     raise "Unsupported finder value #{cnt}" unless cnt == :all
 
-    Vmdb::Deprecation.deprecation_warning(:find, :find_entries)
+    Vmdb::Deprecation.deprecation_warning(:find, :find_entries, caller(1))
```

Comment:
https://github.com/rails/rails/pull/22406

As soon as that's merged/revised/fixed and possibly backported to 4-2-stable, I can update this PR...  Let's see what happens first :wink: